Git-Behind-Proxy.md

```diff
@@ -0,0 +1,38 @@
+# Git Behind A Proxy Server
+
+### Use-cases
+
+You might need to modify `git` commands that accesses (to update them and read from) remote repositories if your internet access is through a [proxy server](https://en.wikipedia.org/wiki/Proxy_server).
+
+It's quite common if you are a student in college, or maybe you are a working professional.
```

Comment:
Maybe reword to something like: 'Proxy servers are common in college and business type environments.'